### PR TITLE
update tyrian to 0.5.0

### PR DESCRIPTION
**Pull request by gvolpe**

### CODE REVIEW

File: modules/ws-client/package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
     "postbuild": "cp -r static/* dist/"
   },
   "devDependencies": {
-    "parcel": "^2.3.2"
+    "parcel": "^2.3.2",
+    "process": "^0.11.10"
   },
   "dependencies": {
     "snabbdom": "^3.0.1"
```
Comment (davesmith00000) [inline modules/ws-client/package.json:12]: @gvolpe You should be able to remove this now. 😄

**Review by davesmith00000 — COMMENTED** [review]
Awesome! Just left a comment about the now redundant npm snabbdom dep.

File: modules/ws-client/package.json (hunk 1 above)
Comment (gvolpe) [reply modules/ws-client/package.json:12]: The `snabbdom` dep, right? Can't remember how to manage `package.json` deps, do I just remove it manually?

Comment (davesmith00000) [reply modules/ws-client/package.json:12]: Yes, so in this case you could just remove the whole `dependencies` section manually, do a `yarn install`  then run it again and it should all still work just fine.

Comment (gvolpe) [reply modules/ws-client/package.json:12]: That did it, thank you!

File: modules/ws-client/package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
     "postbuild": "cp -r static/* dist/"
   },
   "devDependencies": {
-    "parcel": "^2.3.2"
+    "parcel": "^2.3.2",
+    "process": "^0.11.10"
```
Comment (lenguyenthanh) [inline modules/ws-client/package.json:9]: Just wondering why do we need to add this?

Comment (gvolpe) [reply modules/ws-client/package.json:9]: No idea, that was added automagically when I ran the app. Perhaps @davesmith00000 can help answer your question.

Comment (davesmith00000) [reply modules/ws-client/package.json:9]: Couldn't say for sure, but I think it's just a transitive dependency for Parcel.js. However it's only in the `devDependencies` section, so it won't be bundled in your app.

Comment (davesmith00000) [reply modules/ws-client/package.json:9]: But really at this point, as this project stands, all we're using npm/yarn for is managing parcel. So if parcel was installed globally or via a nix-shell or something, then all the work being done here might be done by sbt or Mill instead.

Now of course if Gabriel wanted to start pulling in other npm libs or npm packaged css or soemthing, then we're back to needing this set up or to using something like scalajs-bundler. But... so close to not needing NPM. So close!

Comment (lenguyenthanh) [reply modules/ws-client/package.json:9]: ah, I got it, thanks @gvolpe and @davesmith00000 for your explanation!

**Pull request merged**